Modules/Bridge/NumPy/include/itkPyBuffer.hxx

```diff
@@ -23,136 +23,137 @@
 namespace itk
 {
 
-template<class TImage>
+template < class TImage >
 PyObject *
-PyBuffer<TImage>
-::_GetArrayViewFromImage( ImageType * image)
+PyBuffer< TImage >::_GetArrayViewFromImage( ImageType * image )
 {
-  PyObject *                  memoryView    = NULL;
-  Py_buffer                   pyBuffer;
-  memset(&pyBuffer, 0, sizeof(Py_buffer));
+  PyObject * memoryView = NULL;
+  Py_buffer  pyBuffer;
+  memset( &pyBuffer, 0, sizeof( Py_buffer ) );
 
-  Py_ssize_t                  len           = 1;
-  size_t                      pixelSize     = sizeof(ComponentType);
-  int                         res           = 0;
+  Py_ssize_t len = 1;
+  size_t     pixelSize = sizeof( ComponentType );
+  int        res = 0;
 
-  if( !image )
-    {
-    throw std::runtime_error("Input image is null");
-    }
+  if ( !image )
+  {
+    throw std::runtime_error( "Input image is null" );
+  }
 
   image->Update();
 
-  ComponentType *buffer =  const_cast < ComponentType *> (reinterpret_cast< const ComponentType* > ( image->GetBufferPointer() ) );
+  ComponentType * buffer =
+    const_cast< ComponentType * >( reinterpret_cast< const ComponentType * >( image->GetBufferPointer() ) );
 
   void * itkImageBuffer = (void *)( buffer );
 
   // Computing the length of data
   const int numberOfComponents = image->GetNumberOfComponentsPerPixel();
-  SizeType size = image->GetBufferedRegion().GetSize();
+  SizeType  size = image->GetBufferedRegion().GetSize();
 
-  for( unsigned int dim = 0; dim < ImageDimension; ++dim )
-    {
+  for ( unsigned int dim = 0; dim < ImageDimension; ++dim )
+  {
     len *= size[dim];
-    }
+  }
 
   len *= numberOfComponents;
   len *= pixelSize;
 
-  res = PyBuffer_FillInfo(&pyBuffer, NULL, (void*)itkImageBuffer, len, 0, PyBUF_CONTIG);
-  memoryView = PyMemoryView_FromBuffer(&pyBuffer);
+  res = PyBuffer_FillInfo( &pyBuffer, NULL, (void *)itkImageBuffer, len, 0, PyBUF_CONTIG );
+  memoryView = PyMemoryView_FromBuffer( &pyBuffer );
 
-  PyBuffer_Release(&pyBuffer);
+  PyBuffer_Release( &pyBuffer );
 
   return memoryView;
 }
 
-template<class TImage>
-const typename PyBuffer<TImage>::OutputImagePointer
-PyBuffer<TImage>
-::_GetImageViewFromArray( PyObject *arr, PyObject *shape, PyObject *numOfComponent)
+template < class TImage >
+const typename PyBuffer< TImage >::OutputImagePointer
+PyBuffer< TImage >::_GetImageViewFromArray( PyObject * arr, PyObject * shape, PyObject * numOfComponent )
 {
-  PyObject *                  shapeseq      = NULL;
-  PyObject *                  item          = NULL;
+  PyObject * shapeseq = NULL;
+  PyObject * item = NULL;
 
-  Py_ssize_t                  bufferLength;
-  Py_buffer                   pyBuffer;
-  memset(&pyBuffer, 0, sizeof(Py_buffer));
+  Py_ssize_t bufferLength;
+  Py_buffer  pyBuffer;
+  memset( &pyBuffer, 0, sizeof( Py_buffer ) );
 
-  SizeType size;
-  SizeType sizeFortran;
+  SizeType      size;
+  SizeType      sizeFortran;
```

Comment:
Can we have local variable declarations not aligned on the name? And only keep them for class member variable declarations?

Reply:
https://clang.llvm.org/docs/ClangFormatStyleOptions.html
AlignConsecutiveDeclarations = False

assume 
AlignConsecutiveAssignments = False  also

Reply:
There is no separate setting for local variables vs members variables, so we should leave `AlignConsecutiveDeclarations = True`.